README.md

```diff
@@ -19,61 +19,61 @@ Serenata de Amor is an open project using artificial intelligence for social con
 
 Among founders and long-term members, we can list a group of eight people – plus numerous contributors from the open source and open knowledge communities:  [Tatiana Balachova](https://tatianasb.ru), [Felipe Cabral](https://twitter.com/felipebcabral), [Eduardo Cuducos](https://cuducos.me),  [Irio Musskopf](https://iriomk.com), [Bruno Pazzim](http://brunopazzim.com/), [Ana Schwendler](http://anaschwendler.com/), [Jessica Temporal](http://jtemporal.com/) and [Pedro Vilanova](https://twitter.com/pedrovilanova).
 
 ### How
 
-We use technology just like large organizations such as Google, Facebook and Netflix in order to track government spendings and to make open data accessible for everyone. We started looking into data from the Chamber of Deputies (Brazilian lower house) but we expanded to the Federal Senate (Brazilian upper house) and to municipalities.
+Similar to organizations like Google, Facebook, and Netflix, we use technology to track government spendings and make open data accessible for everyone. We started looking into data from the Chamber of Deputies (Brazilian lower house) but we expanded to the Federal Senate (Brazilian upper house) and to municipalities.
 
 ### When
 
-Irio had the main ideas for the project in early 2016. For a few moths he was experimenting and gathering people around the project. September, 2016 marks the launching of [our first crowd funding](https://catarse.me/serenata). Since then we have been creating open source technological products and tools, as well as high quality content on civic tech on our [Facebook](https://fb.com/operacaoserenatadeamor) and [Medium](https://medium.com/serenata).
+Irio had the main ideas for the project in early 2016. For a few months, he experimented and gathered people around the project. September, 2016 marks the launching of [our first crowd funding](https://catarse.me/serenata). Since then, we have been creating open source technological products and tools, as well as high quality content on civic tech on our [Facebook](https://fb.com/operacaoserenatadeamor) and [Medium](https://medium.com/serenata).
 
 ### Where
 
-We have no non-virtual headquarter, but we work remotely everyday. Most our ideas are crafted to work in any country that offers open data, but our main implementations focus in Brazil.
+We have no non-virtual headquarters, but we work remotely everyday. Most of our ideas are crafted to work in any country that offers open data, but our main implementations focus in Brazil.
 
 ### Why
 
-Empowering citizens with data is quite important nowadays: people talk about _smart cities_, _surveillance_ and _privacy_. We prefer to focus on _smart citizens_, _accountability_ and _open knowledge_.
+Empowering citizens with data is important: people talk about _smart cities_, _surveillance_ and _privacy_. We prefer to focus on _smart citizens_, _accountability_ and _open knowledge_.
```

Comment:
What is the rational for this particular edit here?

Reply:
I think it is generally better to be direct. The use of "quite" makes the sentence softer/more polite, but without it, the sentence is more direct and perhaps clearer. I removed "nowadays" because the sentence already implies that we are talking about the present.

Reply:
100% agree! Many thanks for that : )